science/hdf5/files/patch-no-commons.diff

```diff
@@ -0,0 +1,17 @@
+--- configure.orig	2023-08-10 16:57:56.000000000 -0600
```

Comment:
please add a link jn fhe patchfile to the upstream commit/PR

Reply:
Done.  See corrected PR https://github.com/macports/macports-ports/pull/20572.